src/main/java/com/atlassian/bitbucket/jenkins/internal/model/deployment/BitbucketDeploymentEnvironment.java

```diff
@@ -102,43 +94,10 @@ public int hashCode() {
     @Override
     public String toString() {
         return "BitbucketDeploymentEnvironment{" +
-               "key='" + key + '\'' +
-               ", name='" + name + '\'' +
-               ", type='" + type + '\'' +
-               ", url='" + url + '\'' +
-               '}';
-    }
-
-    public static class Builder {
```

Comment:
As discussed earlier, I felt this builder added nothing of value and just noise. Having seen the actual use of it, it has just confirmed my feeling. Builders on entity objects are a bit weird and unusual anyway.